plugins/inputs/postgresql_extensible/README.md

```diff
@@ -43,6 +43,9 @@ The example below has two queries are specified, with the following parameters:
   # withdbname was true.
   # Be careful that if the withdbname is set to false you don't have to define
   # the where clause (aka with the dbname)
+  # Also script option can be used to speciafy the .sql file path,
```

Comment:
Revise this. Add a space before the block. Plainly document what `script` is for.

Update `SampleConfig()` as well